packages/react/src/components/Tag/Tag-test.js

```diff
@@ -308,4 +308,34 @@ describe('Tag', () => {
     render(<SelectableTag type="red" text="Test Tag" ref={ref} />);
     expect(ref.current).toBeInstanceOf(HTMLButtonElement);
   });
+  it('Controlled selectable tag', () => {
+    const ref = React.createRef();
+    const { rerender, container } = render(
+      <SelectableTag type="red" text="Test Tag" ref={ref} selected={true} />
+    );
+
+    expect(ref.current).toBeInstanceOf(HTMLButtonElement);
+    expect(container.firstChild).toHaveClass(
+      `${prefix}--tag--selectable-selected`
+    );
+
+    rerender(
+      <SelectableTag type="red" text="Test Tag" ref={ref} selected={false} />
+    );
+    expect(container.firstChild).not.toHaveClass(
+      `${prefix}--tag--selectable-selected`
+    );
+  });
+  it('Controlled selectable tag, should call onChange', async () => {
+    const onChange = jest.fn();
+
+    const { container } = render(
+      <SelectableTag text="Tag content" onChange={onChange} selected={true} />
+    );
+
+    const selectableTag = container.firstChild;
+
+    await userEvent.click(selectableTag);
```

Comment:
I'd prefer the tests to use `screen.getByRole` instead of `container.firstChild` and assert `aria-pressed` in addition to the `toHaveClass` assertions (since a user doesn't directly interact with the classnames), but it's not a blocker to merge. 

```suggestion
    await userEvent.click(screen.getByRole("button", {name: "Tag content"}));
```

Reply:
Sure, Updated the tests.